WTWSMS/events/religious_events.txt

```diff
@@ -30,7 +30,15 @@ province_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 9600
+		months = 8000
+		modifier = {
+			factor = 0.15
+			AND = {
+				NOT = { year = 500 }
+				religion = nicene # In the early parts of this Mod, the religious world was in turmoil with the fall of Christian WRE
```

Comment:
Wrong. Nicene does not always take a huge penalty.

I pointed that out in an Issue, that the Miaphysite religion took the Nicene penalty instead, and you laughed it off as Working as Designed, despite that being insanely ahistorical.

As for railroading, you're the one that railraods that extremely zealous Jews cannot Prolestyze and who supports railroading the AI into not being able to form dynamic kingdoms, squashed between prestige requirements that amount to "AI = NO" and draconian de jure restrictions that the player doesn ot receive.